### ISSUE TITLE
ga:activeVisitors not available on query explorer

### ISSUE THREAD

**Issue by johno1985** [googleanalytics/ga-dev-tools#216]
Hi,

I am trying to query the number of active users on our service via the query explorer

![image](https://cloud.githubusercontent.com/assets/1640705/17807675/19109c2e-6604-11e6-8191-1474adc56df3.png)

But as you can it does not recognise that metric, please can you help?

**Comment by mcohoon** [comment]
[`rt:activeUsers`](https://developers.google.com/analytics/devguides/reporting/realtime/dimsmets/user) is a realtime API metric. The Query explorer is built to query the [Core Reporting API V3](https://developers.google.com/analytics/devguides/reporting/core/v3), which can only use those [Dimensions and Metrics](https://developers.google.com/analytics/devguides/reporting/core/dimsmets) which work with the Core Reporting API V3. See the [Help pages](https://developers.google.com/analytics/devguides/reporting/core/v4/help) for details on getting help with the APIs.

## TLDR `ga:activeVisitors` is not a real metric.

**Comment by johno1985** [comment]
Ah, OK, thanks Matt,

Kind regards,

Jonathan.
